src/main.ts

```diff
@@ -74,7 +77,7 @@ export async function run(): Promise<void> {
 
         const config = await Config.load(oclifConfig);
         await new bump.Diff.Diff(config)
-          .run(file1, file2, doc, hub, branch, token, 'markdown', expires)
+          .run(file1, file2, doc, hub, branch, token, 'markdown', expires, overlays)
```

Comment:
Not related but I see that the run of deploy.ts of the CLI as a parameter `overlay` and not `overlays`, (string[])